OpenSim/Simulation/Test/testInitState.cpp

```diff
@@ -50,6 +50,7 @@ int main()
     try {
         LoadOpenSimLibrary("osimActuators");
         testStates("arm26.osim");
+        //TODO: testMemoryUsage fails, is that okay?
```

Comment:
Looks like the test is just not working properly?

>>MODEL: arm26.osim uses 0KB
0KB change in memory use after 100 state initializations.
Approximate leak size: 0KB or -nan% of model size.
Average initialization time: 0.68901ms
testInitState failed: 
Exception:
  testMemoryUsage: state initialization leak > 0.5% of model memory
  footprint.
  file= /home/thomaslau/opensim-core-thomas/OpenSim/Simulation/Test/testInitState.cpp
  line= 196